.github/workflows/release_master.yml

```
@@ -63,6 +63,14 @@ jobs:
           systemd
       - name: Install Deps for installing project
         run: pip install Sphinx coverage wheel
+      - name: Fetch date for version bump
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y%m%d%H%M')"
+      - name: Set env variable for version bump
+        env:
+          NEW_VERSION: ${{ steps.date.outputs.date }}
+      - name: Replace version in setup.py
+        run: sed -i 's/VERSION = "3.2.0"/VERSION = "3.2.0.$NEW_VERSION"/g' setup.py
```

```suggestion
        run: sed -i '/VERSION = "[0-9].[0-9].[0-9]/s/.$/.$NEW_VERSION"/g' setup.py
```

Please check this for yourself @nodeg, I am very uncertain when it comes to `sed`.

---

Alternatively it could also be that we need to do this since we do want that environment variable to be expanded:
```suggestion
        run: sed -i '/VERSION = "[0-9].[0-9].[0-9]/s/.$/.'$NEW_VERSION'"/g' setup.py
```

---

I did not have the time to look into the regex yesterday, so thank you for the suggestion. With your version we are indeed more versatile with future versions. I tested both commands in the terminal and only the second one works as intended.